mlflow/models/evaluation/base.py

```diff
@@ -1623,10 +1690,14 @@ def pred_sample(eval_df, _builtin_metrics, _artifacts_dir):
 
         if isinstance(data, Dataset) and issubclass(data.__class__, PyFuncConvertibleDatasetMixin):
             dataset = data.to_evaluation_dataset(dataset_path, feature_names)
-            if evaluator_name_to_conf_map and evaluator_name_to_conf_map.get("default", None):
-                context = evaluator_name_to_conf_map["default"].get("metric_prefix", None)
-            else:
-                context = None
+
+            # Use metrix_prefix configured for builtin evaluators as a dataset tag
+            context = None
+            for e in evaluators:
```

Comment:
silly nit here, but could we pick a different alias for the evaluator? Maybe it's just me, but everytime I see a plain `e` in code at a glance I always assume it refers to an exception and its stack trace

Reply:
+1

Reply:
maybe `eval_`?

Reply:
Ah that makes sense, what about `ev`?